### ISSUE TITLE
3-finger double tap cause context menu to open

### ISSUE THREAD

**Issue by plmcgrn** [aeharding/voyager#1254]
**Describe the bug**
This is commonly used as an accessibility shortcut.  I use it for full screen zoom.  Ii can't do it with Voyager because it causes the context menu to open in waiting to the zoom.

**To Reproduce**

1. Ena bb Le this in iOS accessibily
2. Open Voyager and then double tap with 3 fingers

**Expected behavior**
Only the screen zoom occurs

**Screenshots**

**Desktop (please complete the following information):**

- OS: iOS

**Smartphone (please complete the following information):**

- Device: iphone 14 Pro and ipad Air 2
- OS: [e.g. iOS8.1] 17.3.1
- Browser [e.g. stock browser, safari] app

**Additional context**

**Comment by aeharding** [comment]
Hi, thanks! I made an issue with the upstream library. If I can't get this resolved in a few weeks, would adding a setting to disable long presses be acceptable?

**Comment by plmcgrn** [comment]
> Hi, thanks! I made an issue with the upstream library. If I can't get this resolved in a few weeks, would adding a setting to disable long presses be acceptable?

Yeah I could live with that.  I only really use that to block communities.
